handler/instance_started.go

```diff
@@ -118,7 +118,12 @@ func (h *InstanceStarted) Handle(ctx context.Context, workerID int, msg kafka.Me
 		"num_variables": len(resp.Dataset.Variables.Edges),
 	})
 
-	ireq := h.createUpdateInstanceRequest(ctx, resp.Dataset.Variables, e, r.CantabularBlob, i.CodeLists)
+	if len(i.Editions) < 1 {
+		return errors.New("no editions found in instance")
+	}
+	edition := i.Editions[0] // for census, we assume there will only ever be one
+
+	ireq := h.CreateUpdateInstanceRequest(ctx, &r.Format, resp.Dataset.Variables, e, r.CantabularBlob, i.CodeLists, edition)
```

Comment:
Why do we need to pass a reference to a string?
Why can't we just pass r.Format?
I would even suggest, that since we are now passing `r.CantabularBlob` and `r.Format` you could pass a copy of a recipe into that function.
```suggestion
	ireq := h.CreateUpdateInstanceRequest(ctx, resp.Dataset.Variables, e, r, i.CodeLists, edition)
```

In this way future dependencies we have on Recipe will not force us to refactor the function signature.

Reply:
Done